keps/0006-stable-kafka-example.md

```diff
@@ -126,4 +131,4 @@ As a Kafka administrator I want to have easy access to all relevant logs so I ca
 
 #### JMX Metrics
 
-As a Kafka administrator I want to have easy access to Kafka's JMX metrics so I can easily debug problems.
\ No newline at end of file
+As a Kafka administrator I want to have easy access to Kafka's JMX metrics so I can easily debug problems.
```

Comment:
this is great. We should also overall think more of how to incorporate in house monitoring and metrics.